ts/editor/editor-toolbar/UnderlineButton.svelte

```diff
@@ -11,7 +11,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     import { getPlatformString } from "../../lib/shortcuts";
     import { getSurrounder } from "../surround";
     import { underlineIcon } from "./icons";
-    import { getNoteEditor } from "../NoteEditor.svelte";
+    import { context } from "../NoteEditor.svelte";
```

Comment:
It's not hard to hover over `context` or jump to its definition to find out what it's referring to, but it might make reading the code a little easier if we doing something like

```
import { context as noteEditorContext } from "../NoteEditor.svelte";
```

It is extra work though, and I don't have strong feelings here. What do you think?

Reply:
I've actually done that in some cases already (esp. when I imported multiple contexts). I'd say the smaller the file, the more forgivable it is :) I'll change it.